### DISCUSSION TITLE
Disable round call on training mode.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1697] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Round call on training mode is pointless and a waste of time.

### Describe the solution you'd like

Pretty much every commercial game gives you control right away on training mode. Would be nice for Ikemen to work like this as well.

**Comment by potsmugen** [comment]
That'll make chars not do their intros though. I think most people would be against it so it doesn't sound like it should be default behavior.

**Reply by UrsoManso** [reply]
What's the point of having intros on training mode tho? But anyway, couldn't the solution be to skip only the round call and give control right after the intro?

**Reply by potsmugen** [reply]
Another quirk I just thought of is that many chars _must_ go through state 5900 to initialize correctly. And state 5900 usually sends them to the intro. So implementing this would mean skipping state 5900 and breaking stuff.

Then again chars could do their intros and just start fighting without a round call.

**Reply by UrsoManso** [reply]
Yeah skipping only the round call sounds like the optimal route to avoid issues.

**Comment by K4thos** [comment, marked as answer]
this is a default behaviour in nightly
